pkgs/applications/networking/browsers/vivaldi/default.nix

```diff
@@ -43,10 +44,13 @@ in stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ patchelf makeWrapper ];
 
+  dontWrapQtApps = true;
```

Comment:
Can we save ourself parts of this file by using the qt wrapper?

Reply:
I don't think so. Vivaldi is not a Qt program and works without it. It just also comes with this `libqt5_shim.so`, which was definitely broken before. I don't know what it is used for but figured it would be better to have it working than not.